src/index.json

```diff
@@ -9697,6 +9697,63 @@
                 "sha256Digest": "a80a939563355331e53c54297710ec3198de460d04d357351a6decee5e6a64d1"
             }
         ],
+        "aksarc": [
+            {
+                "downloadUrl": "https://hybridaksstorage.z13.web.core.windows.net/HybridAKS/CLI/aksarc-1.0.0-py3-none-any.whl",
+                "filename": "aksarc-1.0.0-py3-none-any.whl",
+                "metadata": {
+                    "azext.isExperimental": true,
```

Comment:
This tag is not used any more. For review version, please use version 1.0.0b1

Reply:
We are removing the isExperimental and isPreview flags, I'll update this PR soon

Reply:
Updated it now :)